### DISCUSSION TITLE
next/image error: hostname is not configured under images in your `next.config.js`

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [vercel/next.js#40174] [Help]
I have some images under the `public` folder and I use them in a carousel, but when I start the dev server it gives me an error and I cannot see the carousel
![image](https://user-images.githubusercontent.com/61914365/188057343-3f8ed020-02a7-403e-a29a-44729c3c3821.png)
![image](https://user-images.githubusercontent.com/61914365/188057389-066aa244-166d-484d-b1cd-982e61d2bc0c.png)
my `next.config.js` file
```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  images: {
    domains: ['assets.pokemon.com'],
    formats: ['image/avif', 'image/webp'],
  },
};

module.exports = nextConfig;

```

**Comment by icyJoseph** [comment]
Got a repo to take a look at? Tried to reproduce but it works fine. How are you using the next image for local images? 

I tried with:

- https://nextjs.org/docs/basic-features/image-optimization#local-images
- https://nextjs.org/docs/basic-features/image-optimization#remote-images

No issue found with next@12.2.5

**Reply by Chandler-Zhu** [reply]
the repo: https://github.com/Chandler-Zhu/pkm
I import the images according to the doc, now the errors are gone, but I still cannot see the carousel(it is in the `account` page protected by `next/auth` so you'll have to log in first)
